v3/src/components/map/components/map-point-layer.tsx

```diff
@@ -364,6 +461,9 @@ export const MapPointLayer = observer(function MapPointLayer({mapLayerModel, set
       </svg>
       <div ref={pixiContainerRef} className="map-dot-area"/>
       <MapPointGrid mapLayerModel={mapLayerModel} />
+      <div className="heatmap-area">
+        <canvas ref={heatmapCanvasRef} className="heatmap-canvas" />
+      </div>
```

Comment:
Can we only render the heat map canvas when the heat map is enabled? Otherwise, it just seems like it's cluttering up the DOM.